tac/agents/controller/agent.py

```diff
@@ -108,24 +114,31 @@ def act(self) -> None:
                 logger.debug("[{}]: Checking if we can start the competition.".format(self.name))
                 min_nb_agents = self.game_handler.tac_parameters.min_nb_agents
                 nb_reg_agents = len(self.game_handler.registered_agents)
+
+                # Remove temporary file (temporary measure to communicate)
```

Comment:
@dishmop is this command still relevant?

Reply:
OK thanks for these comments. Yes I can see you have the tac installed rather than running from the directory directly. I've done what I hope is a fix for this now and will commit it. 


I did originally put the .keep file in, but my "clean up" code cleaned it up prior to check-in. I've fixed this too.